applications/nrf_desktop/src/modules/Kconfig.ble

```diff
@@ -160,4 +160,21 @@ source "subsys/logging/Kconfig.template.log_config"
 
 endif
 
+
+comment "BLE Discovery"
+
+config DESKTOP_BLE_DISCOVERY_ENABLE
+	bool "Enable BLE discovery"
+	depends on BT_CENTRAL
```

Comment:
does not depend on DESKTOP_BLE_SCANNING_ENABLE?

Reply:
ok, changed